lang/go/eval/symbol/variety.go

```diff
@@ -56,6 +58,28 @@ type InterpretMacro interface {
 	InterpretFunc() (pkgPath, funcName string)
 }
 
+// DisassembleToGo converts a Ko value into a Go value
+func (vty *VarietySymbol) DisassembleToGo(span *model.Span) (reflect.Value, error) {
+	if pkgPath, funcName, _, err := vty.Dismentle(span); err != nil {
+		return reflect.Value{}, span.Errorf(err, "dismentling variety")
+	} else {
+		/*fields*/ _, err := disassembleFieldSymbolsToGo(span, vty.Arg)
+		if err != nil {
+			return reflect.Value{}, err
+		}
+		/*dis := &pb.SymbolVariety{
+			PkgPath:  proto.String(pkgPath),
+			FuncName: proto.String(funcName),
+			Arg:      fields,
+		}
+		return &pb.Symbol{
+			Symbol: &pb.Symbol_Variety{Variety: dis},
+		}, nil*/
+		return reflect.Value{}, fmt.Errorf("TODO convert variety %s/%s to go", pkgPath, funcName)
```

Comment:
Don't really know what to do with this yet. Suggestions?

Reply:
Return nil. There is no need to throw error.